readme.md

```diff
@@ -418,6 +418,7 @@
 - [Public Datasets](https://github.com/awesomedata/awesome-public-datasets#readme)
 - [Hadoop](https://github.com/youngwookim/awesome-hadoop#readme) - Framework for distributed storage and processing of very large data sets.
 - [Data Engineering](https://github.com/igorbarinov/awesome-data-engineering#readme)
+- [Data Quality](https://github.com/MigoXLab/awesome-data-quality#readme) - Comprehensive collection of data quality resources, tools, papers, and projects across various data types.
```

Comment:
> Your entry here should include a short description of the project/theme of the list. It should not describe the list itself.

I think this entry is too descriptive of the list itself instead of describing what "Data Quality" means as a project/theme.